File: php/tests/unit/FizzBuzzTest.php
<?php

require_once( "fizzbuzz.php" );

class FizzBuzzTest extends \Codeception\TestCase\Test {
    private $fizzbuzz;

    protected function _before() {
        $this->fizzbuzz = new FizzBuzz();
    }

    public function testInputThreeShouldReturnFizz() {
        $result = $this->fizzbuzz->take( 3 );
        $this->assertEquals( "Fizz", $result );
    }

    public function testInputSixShouldReturnFizz() {
        $result = $this->fizzbuzz->take( 6 );
        $this->assertEquals( "Fizz", $result );
    }

    public function testInputFiveShouldReturnBuzz() {
        $result = $this->fizzbuzz->take( 5 );
        $this->assertEquals( "Buzz", $result );
    }
}

File: java/FizzBuzzTest.java
import static org.junit.jupiter.api.Assertions.assertEquals;
import org.junit.jupiter.api.Test;

public class FizzBuzzTest {
    FizzBuzz fizzBuzz = new FizzBuzz();

    @Test
    public void returnsFizzIfGivenThree() {
      String result = this.fizzBuzz.getResult(3);
      assertEquals("Fizz", result);
    }

    @Test
    public void returnsFizzIfGivenSix() {
      String result = this.fizzBuzz.getResult(6);
      assertEquals("Fizz", result);
    }
}
